src/index.ts

```diff
@@ -87,10 +87,10 @@ class AuthProvider {
   /**
    * A function to initialize the wallet, should be called before getting provider
    */
-  public async init(
-    input: InitInput = { appMode: AppMode.NoUI, position: 'right' }
-  ) {
-    const { appMode, position } = input
+  public async init(input?: InitInput) {
+    const appMode = input?.appMode || AppMode.NoUI
```

Comment:
If `input?.appMode` is 0 (which coerces to false), then it will get reset to AppMode.NoUI.

Reply:
`AppMode.NoUI` = `0` so it works out.

Reply:
I guess checking for `undefined` increases readability.

Reply:
Ohk, let me change that then.

Reply:
Does this work? https://github.com/arcana-network/auth/pull/74/commits/77220bf458a2ccc78d496db3e7c7a0afa2b234a8

Reply:
should we check if the value is one of the AppMode and Position?